src/_common/comment/widget/widget.vue

```diff
@@ -64,14 +64,16 @@
 
 			<app-loading v-if="isLoading" class="loading-centered" />
 			<div v-else-if="shouldShowEmptyMessage">
-				<div class="alert alert-info">
-					<template v-if="shouldShowAdd">
-						<translate>It's feeling a bit empty in here. Start talking!</translate>
-					</template>
-					<template v-else>
-						<translate>There are no comments here...</translate>
-					</template>
-				</div>
+				<app-illustration src="~img/ill/no-comments.svg">
+					<p>
+						<translate v-if="shouldShowAdd">
+							It's feeling a bit empty in here. Start talking!
+						</translate>
+						<translate v-else>
+							You're not able to comment on this.
```

Comment:
Why did we change the messaging here?

Reply:
@hworld liked the wording more. The only time we show this message is when there are no comments and the posting user/game has blocked you.

Reply:
I was confused. I didn't know you could see comments still. We changed the messaging.